src/main/java/com/forms/formswebapp/form/FormService.java

```diff
@@ -31,4 +36,15 @@ void updateExpiredForms() {
         expiredForms.forEach(Form::expire);
         formRepository.saveAll(expiredForms);
     }
+
+    FormRequestDto getFormByLink(String link) {
```

Comment:
zrobilbym tutaj juz nowe DTO ktore bedzie sluzylo do zwracania pelnej ankiety w przyszlosci (na razie tylko z linkiem) - FormResponseDto

Reply:
Zmieniłem w kolejnym commicie (tylko mi z jakiegoś powodu zdublowało commita). Możesz zerknąć czy o to chodziło.